fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -196,7 +199,14 @@ def self.example_code
       end
 
       def self.category
-        :beta
+        :deprecated
+      end
+
+      def self.deprecated_notes
+        [
+          "Crashlytics Beta has been deprecated and replaced with Firebase App Distribution.",
```

Comment:
Do we have a hard date for this deprecation? I usually like to include that to show urgency 😊

Reply:
Yeah, it's fast approaching (3/31/2020). I can include it for clarity, but urgency should already be there from other Fabric touch points :)

Reply:
Updated!